form_to_clean_csv.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+import sys
+
+# usage: [form.csv] [hw_number]
+# creates 2 files hw_number_handles.csv & hw_number_teams.csv
+
+data = pd.read_csv(sys.argv[1])
+df = pd.DataFrame(data)
+
+df = pd.DataFrame(data)
+df["Group Name"] = df.index.astype(str) + "_" + df["Group Name"]
+
+df1 = df[["Group Name", "UNI", "GitHub Handle"]]
```

Comment:
don't have a strong opinion about this at all since it doesn't actually matter, but for consistency's sake, should these be `"UNI.0"` and `"GitHub Handle.0"`?

Reply:
this is the column names after the csv file is read i changed the number to 0, 1, and 2 which seems a bit more consistent